javascript/src/main/App.js

```diff
@@ -31,20 +31,20 @@ function App() {
         <Switch>
           <AuthorizedRoute path="/admin" exact component={Admin} authorizedRoles={["admin"]} />
           <AuthorizedRoute path="/admin/slackUsers" exact component={SlackUsers} authorizedRoles={["admin"]} />
-          <AuthorizedRoute path="/admin/teams" exact component={Teams} authorizedRoles={["admin"]} />
-          <AuthorizedRoute path="/admin/searchInfo" exact component={SearchInfo} authorizedRoles={["admin"]} />
```

Comment:
Why did you remove these routes?  I could be mistaken but it looks as if removing those might break other teams' features.

```  
<AuthorizedRoute path="/admin/teams" exact component={Teams} authorizedRoles={["admin"]} />
<AuthorizedRoute path="/admin/searchInfo" exact component={SearchInfo} authorizedRoles={["admin"]} />
```

```
<AuthorizedRoute path="/member/search" exact component={Search} authorizedRoles={["admin","member"]} />
```

```
<AuthorizedRoute path="/member/listViewChannels/:channel" component={ChannelPageList} authorizedRoles={["admin","member"]} />
```

Also you changed the component for 

```
<AuthorizedRoute path="/member/channels/:channel" component={ChannelPageScrollable} authorizedRoles={["admin","member"]} />
```

to 

```
<AuthorizedRoute path="/member/channels/:channel" component={ChannelPage} authorizedRoles={["admin", "member"]} />
```

which, if I'm not mistaken, undoes work by other teams.

Please review this and make sure that each of these changes is necessary and justified.   My thought is that you should likely only be adding the routes below, not changing other teams' routes.  I could be wrong, but we need to be sure:

```
<AuthorizedRoute path="/member/analyzemessages/reactions" exact component={AnalyzeReactions} authorizedRoles={["admin", "member"]} />
          <AuthorizedRoute path="/member/analyzemessages/countmessages" exact component={CountMessagesByUser} authorizedRoles={["admin", "member"]} />
          <AuthorizedRoute path="/member/analyzemessages/messagehistogram" exact component={HistogramOfMessagesByUser} authorizedRoles={["admin", "member"]} />
          <AuthorizedRoute path="/member/analyzemessages/searchmessages" exact component={SearchMessagesByUser} authorizedRoles={["admin", "member"]} />
```

Reply:
Along the same lines, you deleted several files.  Why did you do this?  Do these have to do with your story?

It is important to be sure that you add your code while not interfering with the code of other teams.... I'm not sure, but I'm concerned that your change is touching more code that necessary, and possibly undoing other peoples' work.   Can you help me understand why those files were deleted, and why the routes above were changed?

```
javascript/src/main/components/ChannelMessages/MessageListView.js
javascript/src/main/components/ChannelMessages/time.js
javascript/src/main/pages/Channels/ChannelPageScrollable.js
```

Reply:
I think this happened in the recursive merge of main into my branch since I didn't purposely delete files created by other teams.